ingester/tests/test_storage.py

```diff
@@ -127,19 +129,88 @@ def test_store_conditional_put_latest_multiple_files(dynamodb_latest_table, dyna
     assert len(records) == 2
     assert file2 == res
 
+
+def test_store_conditional_put_newest_first(dynamodb_latest_table, dynamodb_connection):
+    storage = DynamoDBStorage(connection=dynamodb_connection)
+    storage.latest_table_name = 'latest'
+    file1, file2, file3 = provide_test_records()
+
     storage.store_latest(file3)
     storage.store_latest(file2)
     storage.store_latest(file1)
 
-    records = [dict(i) for i in dynamodb_latest_table.scan()]
+    query_what_where = 'syslog:ground_server2'
+
     res = dict(dynamodb_latest_table.get_item(what_where_key=query_what_where))
     assert res['metadata']['id'] != file1['metadata']['id']
     assert res['metadata']['id'] == file2['metadata']['id']
-    
+
+
+def test_store_conditional_put_same_duplicate_overwrite(dynamodb_latest_table, dynamodb_connection):
```

Comment:
What is the behavior this is testing for? What makes it different from `test_store_conditional_put_latest_multiple_files()`?

Reply:
Nothing, it is a duplicate. I'll remove 👍